src/ws/session-utils.c

```diff
@@ -0,0 +1,561 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2013 Red Hat, Inc.
```

Comment:
2019

Reply:
I didn't bump the year as this is just moved code, nothing new. But I don't know if there's some convention about that?

Reply:
Good point. I don't think there's a strong convention either way.